src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -103,6 +103,15 @@ export default (opts) => {
     ),
     incrementsLoader: gravityLoader("increments"),
     inquiryRequestQuestionsLoader: gravityLoader(`inquiry_request_questions`),
+    marketingCollectionLoader: gravityLoader<any, { id: string; size: string }>(
+      ({ id, size }) =>
```

Comment:
This doesn't make a lot of sense to me - shouldn't loading a single marketing collection return it, what's `size`? Follow the examples of other loaders that load a single resource.

Reply:
I see https://github.com/artsy/gravity/pull/17955/files#r1683060386 where this gets used.

Yea...let's not do that 😆 . It's totally atypical. Let's follow existing conventions to return resources - a GET endpoint that returns a single marketing collection, and then a GET endpoint that returns a list of marketing collections which can support returning the related collections to a given one (via `related_to_collection_id` query param for instance).